### DISCUSSION TITLE
Flex Cold transfer - Discoverability on hover of a button

### DISCUSSION THREAD

**Discussion by Porlin** [twilio-labs/paste#1865] [Office hours] [answered]
**Background:** Looking for feedback on the discoverability on hover of a button.
**Stage of design:** [Figma design](https://www.figma.com/file/lVI4BWk6IgBCtJw3EltHwd/Agent-Experience-Conversations?node-id=350%3A107496)
**Research:** To come

**Comment by SiTaggart** [comment]
Hi @Porlin 

As a little pre read, please take a look at the following blog post, [7 Things Every Designer Needs to Know about Accessibility](https://medium.com/salesforce-ux/7-things-every-designer-needs-to-know-about-accessibility-64f105f0881b). 

It does a really good job of explaining the feedback I'll be going into in the office hours, under section 7. Don’t make people hover to find things.

**Comment by Porlin** [comment]
Thanks Simon, I have read over. Thanks for sharing the link.

**Comment by gloriliale** [comment, marked as answer]
Video of this office hours discussion is now [available here for reference!](https://www.loom.com/share/e7c7e600130f4091a41037eb45d3b50e)
